desktop_version/src/DiscordNetwork.c

```diff
@@ -0,0 +1,165 @@
+#include "MakeAndPlay.h"
+
+#ifndef MAKEANDPLAY
+
+#include <stdint.h>
+#include <stdio.h>
+#include <assert.h>
+#include <SDL.h>
+#ifdef _WIN32
+#include <Windows.h>
+#else
+#include <unistd.h>
+#include <string.h>
+#endif
```

Comment:
Are these includes actually used?

Reply:
`assert.h` should be fine to remove but STDInt and STDIO, I'm not sure. And also maybe `string.h` once I use SDL_strncopy

Reply:
I was specifically talking about the `Windows.h` included only if `_WIN32`, and the `unistd.h` and `string.h` which are included only if not `_WIN32`.

Reply:
Well I removed `string.h` and `assert.h` and it now compiles fine.

Reply:
What about `Windows.h` and `unistd.h`?